### DISCUSSION TITLE
Download Unity Package

### DISCUSSION THREAD

**Discussion by khirho** [vrcx-team/VRCX#465] [Q&A]
Question: 

I was exploring through the application and clicked on one of my uploaded avatars. Noticed  it showed "Download Unity Package" 
once I clicked on it  Once downloading it would just show (Avatar-Swift-Unity-package-20184.file_f5dc8f71-(whole lot of numbers).

Q: How exactly do I bring it back to unity in order to edit it?

**Comment by Myrkie** [comment]
If the ending extension for this downloaded package is .unitypackage, then this avatar was uploaded with "future proof publish" enabled within the vrchat SDK. this uploads a full copy of the avatars project to VRChat servers as a .unitypackage, this can be imported directly into unity. if the extension is .vrca then this is a avatars bundle file and that would require ripping which is no something we do here.

**Comment by khirho** [comment]
<img width="371" alt="Screenshot 2023-01-01 173307" src="https://user-images.githubusercontent.com/33561590/210178019-0b5ca7fc-5fbd-456d-8527-398f0c766694.png">


let me show a screenshot but it does not show either,  it just shows it as a .file

**Comment by Myrkie** [comment]
try renaming its extension to .unitypackage if its actually a unity package them you should be able to double click on it or drag drop it into your assets folder in unity to import it.
